packages/edit-post/src/components/keyboard-shortcut-help-modal/test/__snapshots__/index.js.snap

```diff
@@ -1,106 +1,381 @@
-          "description": "Remove a link.",
-          "keyCombination": Object {
-            "character": "k",
-            "modifier": "primaryShift",
-          },
-        },
-        Object {
-          "description": "Insert a link to a post or page",
-          "keyCombination": Object {
-            "character": "[[",
-          },
-        },
-        Object {
-          "description": "Underline the selected text.",
-          "keyCombination": Object {
-            "character": "u",
-            "modifier": "primary",
-          },
-        },
-        Object {
-          "description": "Strikethrough the selected text.",
-          "keyCombination": Object {
-            "character": "d",
-            "modifier": "access",
-          },
-        },
-        Object {
-          "description": "Make the selected text inline code.",
-          "keyCombination": Object {
-            "character": "x",
-            "modifier": "access",
-          },
-        },
-      ]
-    }
-    title="Text formatting"
-  />
-</ForwardRef(Modal)>
+  <div
+    class="components-modal__content"
+    role="document"
+  >
+    <div
+      class="components-modal__header"
+    >
+      <div
+        class="components-modal__header-heading-container"
+      >
+        <h1
+          class="components-modal__header-heading"
+          id="components-modal-header-0"
+        >
+          Keyboard shortcuts
+        </h1>
+      </div>
+      <button
+        aria-label="Close dialog"
+        class="components-button has-icon"
+        type="button"
+      >
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          height="24"
+          viewBox="0 0 24 24"
+          width="24"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"
+          />
+        </svg>
+      </button>
+    </div>
+    <section
+      class="edit-post-keyboard-shortcut-help-modal__section edit-post-keyboard-shortcut-help-modal__main-shortcuts"
+    >
+      <ul
+        class="edit-post-keyboard-shortcut-help-modal__shortcut-list"
+        role="list"
+      >
+        <li
+          class="edit-post-keyboard-shortcut-help-modal__shortcut"
+        />
+      </ul>
+    </section>
+    <section
+      class="edit-post-keyboard-shortcut-help-modal__section"
+    >
+      <h2
+        class="edit-post-keyboard-shortcut-help-modal__section-title"
+      >
+        Global shortcuts
+      </h2>
+      <ul
+        class="edit-post-keyboard-shortcut-help-modal__shortcut-list"
+        role="list"
+      />
+    </section>
+    <section
+      class="edit-post-keyboard-shortcut-help-modal__section"
+    >
+      <h2
+        class="edit-post-keyboard-shortcut-help-modal__section-title"
+      >
+        Selection shortcuts
+      </h2>
+      <ul
+        class="edit-post-keyboard-shortcut-help-modal__shortcut-list"
+        role="list"
+      />
```

Comment:
Same

Reply:
Fixed as part of 4e5e1a0c7fd8aadf12b03357b97f948a8bdd92bb.